app/components/course/corse.component.ts

```diff
@@ -17,13 +17,13 @@ import { Component, Input } from '@angular/core';
                 </div>
             </div>
             <div class="btnContainer">
-                <icon-edit></icon-edit>
-                <icon-delete></icon-delete>
+                <icon-edit (click)="edit()"></icon-edit>
+                <icon-delete (click)="delete()"></icon-delete>
             </div>
         </div>
     `,
     styles: [`
-        :host {
+        :host {  
```

Comment:
Have you used newly added tslint? 😉

Reply:
fixed tslint warnings

Reply:
Why does this line still have trailing spaces then?..

Reply:
Why do you still have trailing whitespace on this line?